k8s/secrets-backup/service-accounts.yaml

```
@@ -0,0 +1,33 @@
+---
+apiVersion: v1
+kind: ServiceAccount
+metadata:
+  name: secrets-backup
+  namespace: custom
+  annotations:
+    eks.amazonaws.com/role-arn: arn:aws:iam::588562868276:role/SecretsBackupRole-spack-production
```

```suggestion
    eks.amazonaws.com/role-arn: arn:aws:iam::588562868276:role/SecretsBackupRole-production
```

See https://github.com/spack/spack-infrastructure/blob/main/terraform/production/main.tf#L81 (alternatively, the name of the IAM role can be changed in the TF code)

---

Whoops, confused `deployment-name` with the cluster name. Good catch.